packages/config/config/devices/0x0258/Motion Sensor 3

```diff
@@ -0,0 +1,174 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "Motion Sensor 3",
+	"description": "PIR Motion Sensor 3",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x6082"
```

Comment:
I think this is supposed to be `0x608d`?

Reply:
Yes it should - typo.